moment-recur.js

```diff
@@ -252,6 +268,10 @@
             if (rule.measure === 'weeksOfMonthByDay' && !this.hasRule('daysOfWeek')) {
```

Comment:
Here we also have measure ```weeksOfMonthByDay``` so that 'toLowercase' rule seems a bit arbitrary and bogus.